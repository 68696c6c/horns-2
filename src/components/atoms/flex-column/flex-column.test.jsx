/* global describe, it, expect */
import React from 'react'

import { render } from 'setup-test'

import FlexColumn from '.'

describe('FlexColumn', () => {
  it('should render as default', () => {
    const { container } = render(
      <FlexColumn>
        <span>example 1</span>
        <span>example 2</span>
        <span>example 3</span>
      </FlexColumn>
    )
    expect(container).toMatchSnapshot()
  })
})
