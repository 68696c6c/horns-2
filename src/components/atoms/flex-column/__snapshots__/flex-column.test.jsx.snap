// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlexColumn should render as default 1`] = `
.emotion-0 {
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: space-evenly;
  -webkit-justify-content: space-evenly;
  -ms-flex-pack: space-evenly;
  justify-content: space-evenly;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: block;
}

@media (min-width:480px) {
  .emotion-0 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
  }
}

.emotion-0 > * {
  -webkit-flex-shrink: 1;
  -ms-flex-negative: 1;
  flex-shrink: 1;
  -webkit-box-flex: 0;
  -webkit-flex-grow: 0;
  -ms-flex-positive: 0;
  flex-grow: 0;
  -webkit-flex-basis: auto;
  -ms-flex-preferred-size: auto;
  flex-basis: auto;
}

<div>
  <div
    class="emotion-0"
    x="center"
    y="evenly"
  >
    <span>
      example 1
    </span>
    <span>
      example 2
    </span>
    <span>
      example 3
    </span>
  </div>
</div>
`;
